contracts/src/interfaces/IEigenDAStructs.sol

```diff
@@ -82,8 +82,8 @@ struct BlobHeaderV2 {
     uint16 version;
     bytes quorumNumbers;
     BlobCommitment commitment;
-    uint32 salt;
     bytes32 paymentHeaderHash;
+    uint32 salt;
```

Comment:
Isn't this struct being called `BlobHeaderV2` a bit limitating? What if we ever in the future want to also reveal payments onchain? Then we'll have to create another struct with a weird name for that.

Perhaps we could keep `BlobHeaderV2` for the full struct to match offchain code, and use a more specific name here for this struct, like `BlobHeaderV2WithPaymentHash` (there's prob a better name)